kernel/kselftest.py

```diff
@@ -92,10 +93,19 @@ def test(self):
         """
         Execute the kernel selftest
         """
+        self.count = 0
```

Comment:
Two points here:
- You're not using `self.count` outside `test()`, so no need for it to be an object attribute. A local variable is enough. So, you could use `count` instead of `self.count`
- You're using `self.count` to make a boolean-like test in line 107. I mean, you're not really using its value other than to check if it's 0 or non-0. In that case, you could call it `errors` instead of `count` and initialize it as `False`, changing it to `True` when some error happens.